projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -43,27 +48,46 @@ TUFTOOL_TARGET=$(CARGO_HOME)/bin/tuftool
 BOTTLEROCKET_SETUP_TARGET=$(BOTTLEROCKET_DOWNLOAD_PATH)/bottlerocket-root-json-checksum
 
 FINAL_UBUNTU_OVA_PATH=$(ARTIFACTS_PATH)/ubuntu.ova
+FINAL_UBUNTU_RAW_IMAGE_PATH=$(ARTIFACTS_PATH)/ubuntu.gz
 FINAL_BOTTLEROCKET_OVA_PATH=$(ARTIFACTS_PATH)/bottlerocket.ova
 FAKE_UBUNTU_OVA_PATH=$(IMAGE_BUILDER_DIR)/output/fake-ubuntu.ova
-
-BUILD_TARGETS=build-ami-ubuntu-2004 release-ova-bottlerocket $(FAKE_UBUNTU_OVA_PATH) upload-artifacts
-RELEASE_TARGETS=release-ova-ubuntu-2004 release-ova-bottlerocket upload-artifacts
+FAKE_UBUNTU_RAW_PATH=$(IMAGE_BUILDER_DIR)/output/fake-ubuntu.gz
+
+BUILD_TARGETS=$(BUILD_RAW_TARGETS) $(BUILD_AMI_TARGETS) $(BUILD_OVA_TARGETS)
+BUILD_AMI_TARGETS=build-ami-ubuntu-2004
+BUILD_OVA_TARGETS=release-ova-bottlerocket $(FAKE_UBUNTU_OVA_PATH) upload-artifacts
+BUILD_RAW_TARGETS=release-raw-ubuntu-2004 $(FAKE_UBUNTU_RAW_PATH) upload-artifacts
+RELEASE_TARGETS=release-ova-ubuntu-2004 release-ova-bottlerocket
+S3_TARGET_PREREQUISITES=$(FINAL_UBUNTU_OVA_PATH) $(FINAL_BOTTLEROCKET_OVA_PATH)
+ifeq ($(IMAGE_FORMAT),raw)
```

Comment:
I dont love all this stuff... i dont have any great alternative tho. I am assuming we are doing this to support building these via separate builds instead of all at once?

Reply:
Yes we have separate build infra per image type (ami, ova, raw), so the raw image builds will run in their own pipeline.